core/embed/bootloader/protob/messages.pb.c

```diff
@@ -1,5 +1,5 @@
 /* Automatically generated nanopb constant definitions */
-/* Generated by nanopb-0.4.0-dev */
+/* Generated by nanopb-0.3.9.4 */
```

Comment:
just out of curiosity, why did we _down_-grade nanopb?

Reply:
I used the master branch in the past to generate these files, but what I should've used was the maintenance_0.3 branch. The code is almost the same in these two branches, but versions are different.